modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving HA components
+
+Active/passive high availability (HA) is now available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
```

Comment:
kourier-gateway is actually active/active

Reply:
Just removed Active/passive entirely to simplify.